lib/constants/dependency.ts

```diff
@@ -0,0 +1,33 @@
+// Dependency Type
+export const DEP_TYPE_CONTAINER_PULL = 'container_pull';
+export const DEP_TYPE_DEPENDENCY = 'dependencies';
```

Comment:
Not 100% on the naming of this. Same for `devDependencies`, `peerDependencies`, and `optionalDependencies`.

Reply:
Yeah, I was also unsure. In that case, spell them out? e.g. `DEP_TYPE_DEPENDENCIES`, etc?

Reply:
Probably the best option.